configure.in

```
@@ -5013,6 +5013,25 @@ if test "$wxUSE_FSWATCHER" = "yes"; then
     fi
 fi
 
+dnl ---------------------------------------------------------------------------
+dnl xkbcommon library for key code translations on Linux/BSD
+dnl ---------------------------------------------------------------------------
+if test "$wxUSE_GTK" = 1; then
+    if test "$USE_WIN32" != 1 ; then
+        if test "$USE_DARWIN" != 1 ; then
+            PKG_CHECK_MODULES(XKBCOMMON, [xkbcommon],
+                [
+                    CFLAGS="$XKBCOMMON_CFLAGS $CFLAGS"
+                    CXXFLAGS="$XKBCOMMON_CFLAGS $CXXFLAGS"
+                    GUI_TK_LIBRARY="$GUI_TK_LIBRARY $XKBCOMMON_LIBS"
+                    AC_DEFINE(HAVE_XKBCOMMON, 1, [Define if libxkbcommon is available]) have_xkbcommon=yes
```

I think this could be just
```suggestion
                    AC_DEFINE(HAVE_XKBCOMMON)
```
because we don't care about the comment (this is used with autoheader which we don't use) and `have_xkbcommon` just doesn't seem to be used at all.